src/pages/Dashboard/EventReadOnly/EventReadOnly.jsx

```diff
@@ -265,60 +269,116 @@ function EventReadOnly() {
+                                moment.tz(
+                                  eventData?.startDate ?? eventData?.startDateTime,
+                                  eventData?.scheduleTimezone ?? 'Canada/Eastern',
+                                ),
+                                moment.tz(
+                                  eventData?.endDate ?? eventData?.endDateTime,
+                                  eventData?.scheduleTimezone ?? 'Canada/Eastern',
+                                ),
+                              ]}
+                              bordered={false}
+                              allowClear={false}
+                              inputReadOnly={true}
+                              open={false}
+                              suffixIcon={false}
+                              // style={{ width: '423px' }}
+                            />
+                          </div>
+                        </Col>
+                      </Row>
                     </>
                   )}
                   <br />
                   <Row justify="space-between">
-                    {eventData?.startDateTime && (
-                      <Col>
-                        <p className="read-only-event-content-sub-title-primary">
-                          {t('dashboard.events.addEditEvent.dates.startTime')}
-                        </p>
-                        <p className="read-only-event-content">
-                          {moment
-                            .tz(eventData?.startDateTime, eventData?.scheduleTimezone ?? 'Canada/Eastern')
-                            .format('h:mm a')}
-                        </p>
-                      </Col>
-                    )}
-                    {eventData?.endDateTime && (
+                    {eventData?.startDateTime &&
+                      eventData?.recurringEvent?.frequency !== dateFrequencyOptions[2].value && (
+                        <Col>
+                          <p className="read-only-event-content-sub-title-primary">
+                            {t('dashboard.events.addEditEvent.dates.startTime')}
+                          </p>
+                          <p className="read-only-event-content">
+                            {moment
+                              .tz(eventData?.startDateTime, eventData?.scheduleTimezone ?? 'Canada/Eastern')
+                              .format(i18n?.language === 'en' ? 'h:mm a' : 'HH:mm')}
+                          </p>
+                        </Col>
+                      )}
+                    {eventData?.endDateTime && eventData?.recurringEvent?.frequency !== dateFrequencyOptions[2].value && (
                       <Col>
                         <p className="read-only-event-content-sub-title-primary">
                           {t('dashboard.events.addEditEvent.dates.endTime')}
                         </p>
                         <p className="read-only-event-content">
                           {moment
```

Comment:
Not urgent, but in the future I recommend creating your own component for these types of configurations so they are only configured in one place. For example the format based on 'en' should ideally only be in one spot. Also, you could remove the "moment" dependency from this jsx file completely and regroup all your moment configurations.  Just a suggestion for future versions, so please don't make too many changes during this gold candidate stage :-)